src/client/stylesheets/style.less

```diff
@@ -191,6 +189,12 @@ body {
 	.picture-fallback-icon(800%);
 }
 
+/* Initial styling for FontAwesome icons */
+.svg-inline--fa{
+	height: 1em;
+	vertical-align: -0.125em;
+}
+
```

Comment:
I think this is redundant.
I inspected the elements in chrome, and found that there is already css for that class that does the same thing (the bottom one):

<img width="248" alt="Capture d’écran 2019-11-12 à 16 28 59" src="https://user-images.githubusercontent.com/6179856/68685145-f0ed2600-0569-11ea-8750-a7f31aab373d.png">

Reply:
![icons](https://user-images.githubusercontent.com/37334979/68688080-2c9ce580-0594-11ea-8f97-d6dd498aee1d.png)
This was the issue and I wrote that as a temporary fix.

Reply:
Yes, I just tried it locally, pretty intense :D

Reply:
Should I include it in the commit msg (it was in the first commit) or is there any better solution?

Reply:
I don't see a better solution, I think that was the correct way to solve this [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content).

I would like the comment for that less block to have just a little bit more context so that it doesn't get deleted down the line (for example by future me :) ) looking for cleanup/refactoring tasks.

Reply:
> https://github.com/bookbrainz/bookbrainz-site/pull/317/files#diff-15715f45efc23cafade2274437cbea8dR31-R32

This will prevent the override!